tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -146,14 +146,11 @@ addr = "172.16.31.10:8287"
 
 # Whether to detect and resolve duplicate records (unique key conflict) in the physical import mode.
 # The following resolution algorithms are supported:
-#  - record: After the data is written to the target table,
-#    add the duplicate records from the target table to the `lightning_task_info.conflict_error_v1` table in the target TiDB.
-#    Note that the required version of the target TiKV is no earlier than v5.2.0;
-#    otherwise it falls back to 'none'.
 #  - none: does not detect duplicate records, which has the best performance of the three algorithms.
 #    But if there are duplicate records in the data source, it might lead to inconsistent data in the target TiDB.
-#  - remove: records all duplicate records in the target table to the lightning_task_info database, like the 'record' algorithm.
-#    But it removes all duplicate records from the target table to ensure a consistent state in the target TiDB.
+#  - remove: If there are primary key or unique key conflicts between the inserting data A and the existing data B in the target table, the conflicting data A and B will be removed from the target table and recorded in the lightning_task_info.conflict_error_v1 table in the target TiDB.   Users can manually reinsert the correct records into the target table based on their business requirements. 
+#    Note that the required version of the target TiKV is no earlier than v5.2.0;
+#    otherwise it falls back to 'none'.
```

Comment:
```suggestion
#  - remove: If there are primary key or unique key conflicts between the inserting data A and the existing data B in the target table, A and B will be removed from the target table and recorded in the `lightning_task_info.conflict_error_v1` table in the target TiDB. You can manually insert the correct records into the target table based on your business requirements. 
# Note that the target TiKV must be v5.2.0 or later versions; otherwise it falls back to 'none'.
```